app/site/_includes/project-type-tag.liquid

```diff
@@ -0,0 +1,14 @@
+  {% when "stadium" %}
+    <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="132" height="20" role="img" aria-label="project type: {{ type }}"><title>project type: {{ type }}</title><linearGradient id="s" x2="0" y2="100%"><stop offset="0" stop-color="#bbb" stop-opacity=".1"/><stop offset="1" stop-opacity=".1"/></linearGradient><clipPath id="r"><rect width="132" height="20" rx="3" fill="#fff"/></clipPath><g clip-path="url(#r)"><rect width="77" height="20" fill="#555"/><rect x="77" width="55" height="20" fill="{{ color }}"/><rect width="132" height="20" fill="url(#s)"/></g><g fill="#fff" text-anchor="middle" font-family="Verdana,Geneva,DejaVu Sans,sans-serif" text-rendering="geometricPrecision" font-size="110"><text aria-hidden="true" x="395" y="150" fill="#010101" fill-opacity=".3" transform="scale(.1)" textLength="670">project type</text><text x="395" y="140" transform="scale(.1)" fill="#fff" textLength="670">project type</text><text aria-hidden="true" x="1035" y="150" fill="#010101" fill-opacity=".3" transform="scale(.1)" textLength="450">{{ type }}</text><text x="1035" y="140" transform="scale(.1)" fill="#fff" textLength="450">{{ type }}</text></g></svg>
+  {% when "federation" %}
+    <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="144" height="20" role="img" aria-label="project type: {{ type }}"><title>project type: {{ type }}</title><linearGradient id="s" x2="0" y2="100%"><stop offset="0" stop-color="#bbb" stop-opacity=".1"/><stop offset="1" stop-opacity=".1"/></linearGradient><clipPath id="r"><rect width="144" height="20" rx="3" fill="#fff"/></clipPath><g clip-path="url(#r)"><rect width="77" height="20" fill="#555"/><rect x="77" width="67" height="20" fill="{{ color }}"/><rect width="144" height="20" fill="url(#s)"/></g><g fill="#fff" text-anchor="middle" font-family="Verdana,Geneva,DejaVu Sans,sans-serif" text-rendering="geometricPrecision" font-size="110"><text aria-hidden="true" x="395" y="150" fill="#010101" fill-opacity=".3" transform="scale(.1)" textLength="670">project type</text><text x="395" y="140" transform="scale(.1)" fill="#fff" textLength="670">project type</text><text aria-hidden="true" x="1015" y="150" fill="#010101" fill-opacity=".3" transform="scale(.1)" textLength="570">{{ type }}</text><text x="1095" y="140" transform="scale(.1)" fill="#fff" textLength="570">{{ type }}</text></g></svg>
+  {% else %}
+    <img src="{{ link }}" alt="Project Type"/>
```

Comment:
For `nadia_shields_badge_url` in the metrics data, can you remove the "#" from the color param? It isn't setting the color accordingly because of it